packages/framework-provider-kubernetes-infrastructure/package.json

```diff
@@ -24,7 +24,13 @@
     "cors": "^2.8.5",
     "express": "^4.17.1",
     "form-data": "^3.0.0",
-    "isomorphic-fetch": "^2.2.1"
+    "isomorphic-fetch": "^2.2.1",
+    "archiver": "4.0.1",
+    "mustache": "4.1.0",
+    "axios": "0.21.1",
```

Comment:
Why do you need Axios?

Reply:
If I recall correctly we are usong axios at runtime in the index.js that the pods are using with express

Reply:
I asked that because I don't remember adding Axios dependency to my Booster project to make it work. If you see it's necessary, of course leave it

Reply:
Yeah, just remove axios :)